graphql/admin/admin.go

```diff
@@ -408,22 +408,24 @@ func newAdminResolver(
 
 func newAdminResolverFactory() resolve.ResolverFactory {
 
-	adminMutationResolvers := map[string]resolve.MutationResolverFunc{
-		"backup":   resolveBackup,
-		"config":   resolveConfig,
-		"draining": resolveDraining,
-		"export":   resolveExport,
-		"login":    resolveLogin,
-		"restore":  resolveRestore,
-		"shutdown": resolveShutdown,
+	adminMutationResolvers := map[string]resolve.MutationResolver{
```

Comment:
Please make this configuration into a map (or even a configuration file).

"backup": applyMiddleWare(resolveBackup, middleware["backups"])

Reply:
```go
middleware := {
	"backup": &MiddlewareOptions{
		poorman: true,
		ipwhitelist: true
	}
}

http.handle("/backup", AdminAuthHandler(handleBackup, middleware["backups"]))
foo.AddMutationResolver(AdminAuthResolver(someResolver(qtx, etc...), middleware["backups"]))
```